composables/useBase.ts

```diff
@@ -0,0 +1,182 @@
+    const rootFullName = computed<string>(() => `${rootName.value} ${rootSurname.value}`)
+    const rootUserEmail = computed(() =>
+        isConsumer.value ? consumer.value?.rootUser.email : corporate.value?.rootUser.email
+    )
+
+    const accountProfileId = computed<string>(() =>
+        isConsumer.value
+            ? root!.$config.profileId.managed_accounts_consumers!
+            : root!.$config.profileId.managed_accounts_corporates!
+    )
+
+    const cardProfileId = computed(() =>
+        isConsumer.value
+            ? root!.$config.profileId.managed_cards_consumers!
+            : root!.$config.profileId.managed_cards_corporates!
+    )
+
+    const profileBaseCurrency = computed(() =>
+        isConsumer.value ? consumer.value?.baseCurrency : corporate.value?.baseCurrency || null
+    )
+
+    const rootName = computed<string>(() => {
+        if (isConsumerAndPopulated.value) {
+            return consumer.value!.rootUser.name
+        } else if (isCorporateAndPopulated.value) {
+            return corporate.value!.rootUser.name
+        }
+
+        return 'noname'
+    })
+
+    const rootSurname = computed<string>(() => {
+        if (isConsumerAndPopulated.value) {
+            return consumer.value!.rootUser.surname
+        } else if (isCorporateAndPopulated.value) {
+            return consumer.value!.rootUser.surname
+        }
+
+        return 'nosurname'
+    })
+
+    const countiesOptions = computed(() => {
+        return Countries.map((_c) => {
+            return {
+                text: _c.name,
+                value: _c['alpha-2'],
+            }
+        })
+    })
+
+    const mobileCountries = computed<string[]>(() => {
+        return Countries.map((_c) => {
+            return _c['alpha-2']
+        })
+    })
+
+    const countryOptionsWithDefault = computed(() => {
+        const _default = [{ ...DefaultSelectValueConst }]
+        _default.push(...countiesOptions.value)
+
+        return _default
+    })
+
+    const identityVerified = computed<boolean>(() => {
+        if (isConsumer.value) {
+            return consumerKyCStatus.value === KYCStatusEnum.APPROVED
+        }
+
+        return corporateKyBStatus.value === KYBStatusEnum.APPROVED
+    })
+
+    function sleep(ms: number) {
+        return new Promise((resolve) => setTimeout(resolve, ms))
+    }
+
+    function goToIndex() {
+        return root!.$router.push('/')
+    }
+
+    function goToVerify() {
+        return root!.$router.push({
+            path: '/login/verify',
+            query: {
+                email: rootUserEmail.value,
+                send: 'true',
+            },
+        })
+    }
+
+    function doLogout() {
+        return stores.auth.logout().then(redirectToLogin)
+    }
+
+    function redirectToLogin() {
+        root!.$router.push('/login')
+    }
+
+    function showSuccessToast(msg?: string, title?: string) {
+        return root!.$weavrToast(msg !== undefined ? msg : 'All changes have been saved', {
```

Comment:
```suggestion
        return root!.$weavrToast(msg ? msg : 'All changes have been saved', {
```

Shouldn't tis be enough?